modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-test/src/testFunctional/tests/CPReportsadmin.testcase

```diff
@@ -264,36 +264,36 @@ definition {
 		ReportsAdmin.gotoDefinitions();
 
 		ReportsAdmin.addReportDefinition(
-			dataSource = "Portal",
+			dataSource = "PORTAL",
 			reportsDefinitionDescription = "Reports Definition Description",
 			reportsDefinitionName = "Reports Definition Name",
 			templateFile = "reports_admin_template_source_sample_class_name.jrxml");
 
 		ReportsAdmin.addReportEntry(
 			emailNotificationsAddress = "test@liferay.com",
 			emailRecipientAddress = "test@liferay.com",
-			reportFormat = "pdf",
+			reportFormat = "PDF",
 			reportsDefinitionName = "Reports Definition Name");
 
 		ReportsAdmin.viewReportEntry(reportsDefinitionName = "Reports Definition Name");
 
 		ReportsAdmin.viewReportEntryFields(
-			dataSource = "portal",
+			dataSource = "PORTAL",
 			reportsDefinitionDescription = "Reports Definition Description",
 			reportsDefinitionName = "Reports Definition Name",
-			reportsFileName = "reports_admin_template_source_sample_class_name.pdf",
+			reportsFileName = "reports_admin_template_source_sample_class_name.PDF",
```

Comment:
@JanisZhang The file name with ".PDF" is kind of weird, please check why we need to use upper case here, let's try to figure out a way to also accept lower case.

Tina